src/os/bluestore/bluestore_tool.cc

```diff
@@ -458,7 +459,7 @@ int main(int argc, char **argv)
     }
   }
 
-  if (action == "fsck" || action == "repair" || action == "quick-fix" || action == "allocmap" || action == "qfsck" || action == "restore_cfb") {
+  if (action == "fsck" || action == "repair" || action == "quick-fix" || action == "allocmap" || action == "qfsck" || action == "restore_cfb" || action == "downgrade-wal-v1") {
```

Comment:
"downgrade-wal-v1" feels like you do something on wal-v1.
"downgrade-to-wal-v1" ?
"wal-v1" ?
Also, need to update documentation.